cooperator_portal/controllers/portal.py

```diff
@@ -76,12 +76,18 @@ def _prepare_home_portal_values(self, counters):
         return values
 
     def _get_invoices_domain(self):
-        return super()._get_invoices_domain() + [
-            ("release_capital_request", "=", False)
-        ]
+        capital_release_request = request.context.get("capital_release_requests", False)
```

Comment:
this method is used in `odoo.addons.account.controllers.portal.PortalAccount` (in `_prepare_home_portal_values()` and `_prepare_my_invoices_values()`), and using the context allows to avoid overriding these methods. with this the capital release requests are excluded from the invoice count and list. do you have a better solution in mind?

i’ll add a comment too, as it’s not obvious.